lib/serve/ecs-model/vllm/src/entrypoint.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -e
+
+declare -a vars=("S3_BUCKET_MODELS" "LOCAL_MODEL_PATH" "MODEL_NAME" "S3_MOUNT_POINT" "THREADS")
+
+# Check the necessary environment variables
```

Comment:
Are these variables always apart of these base images? or is this something that we should make sure we add to the README?

Reply:
These are provided from the container build params within the typescript definition, so they should all be defined as part of this block: https://github.com/awslabs/LISA/blob/646e0fa057a5c6aa76acb7acf151b4b58b649ae1/lib/serve/ecs-model/index.ts#L86-L123